api/src/utils/media/dwc/dwc-archive-file.ts

```diff
@@ -85,6 +85,22 @@ export class DWCArchive {
     }
   }
 
+  /**
+   * This function checks worksheet data if the DwCArchive only contains metadata
+   *
+   * @returns {boolean} True if no worksheet data is present
+   */
+  isMetaDataOnly(): boolean {
+    return (
+      this.eml != null &&
+      this.worksheets.event == null &&
```

Comment:
Should we be using triple-equals instead, to check if these are strictly equal to null? If we're just checking that these properties are truthy and not if they're strictly null, maybe you could just use `!this.worksheets.event && ...`.

Reply:
good point. I've refined the checks